README.md

```diff
@@ -1,2 +1,122 @@
-# setup-tektoncd
-GitHub Action to rollout Tekton Pipelines and auxiliary components of the ecosystem
+[![use-action][useActionWorkflowBadge]][useActionWorkflow]
+
+`setup-tektoncd`
+----------------
+
+Action to rollout [Tekton Pipeline][githubTektonPipeline], [CLI (`tkn`)][githubTektonCLI] and a [Container-Registry][containerRegistry] instance, setting up the environment for testing using these components.
+
+# GitHub Action Usage
+
+Example usage below with inputs using default values:
+
+```yaml
+---
+jobs:
+  setup-tektoncd:
+    steps:
+      # using KinD to provide the Kubernetes instance and kubectl
+      - uses: helm/kind-action@main
```

Comment:
Any reason to "use" `main` here instead of 1.5.0 (as it seems to be the latest released one) ?

Reply:
Indeed, the latest release is better here. Let me change it.

Reply:
Please consider latest changes.